sql/updates/auth/2015_11_08_00_auth.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE `account` DROP COLUMN `mutetime`, DROP COLUMN `mutereason`, DROP COLUMN `muteby`; 
+ALTER TABLE `account_muted` CHANGE `guid` `account` INT(10) UNSIGNED DEFAULT 0 NOT NULL COMMENT 'Global Unique Identifier';
```

Comment:
I appreciate the effort you have put into this PR to simplify how the account mute information is handled. It is just this one single change that I find a bit uncertain: the word "account" is not very specific by itself. In its widest sense, it can mean everything connected to an account (`name`,`id`,`guid` etc.). How about naming it `account_guid` to make sure it is specific and precise enough to be understood, even when taken out of context? Notice that the comment text ('Global Unique Identifier') is still referring to this column as if the name was unchanged.